### DISCUSSION TITLE
RSS feed is broken :(

### DISCUSSION THREAD

**Discussion by Shrinks99** [coldtype/coldtype#163] [General]
Hi Rob!  I like your thoughts!  I want them injected straight into my RSS feed reader, but right now the feed seems broken and all I get is the first post with a summary line :(

I've been on a big RSS feed kick recently.  Happy to help fix this (if I am able) but I couldn't find the site code for the Coldtype blog in the repo?

**Comment by stenson** [comment]
Hi Henry — thanks for starting this discussion. The blog code was previously in a private repo, though I've moved it into the main coldtype repo now (https://github.com/coldtype/coldtype/tree/main/docs/blog) — it's a little obscure how to run, but hopefully the feed.xml (https://blog.coldtype.xyz/feed.xml) should be better now.

**Reply by Shrinks99** [reply]
Heyyyyyy very cool!  Works in reader view with NetNewswire (loads the actual page), I'll see about getting the post content in there this weekend unless you get to it first or if I can't figure it out :)
